core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -267,7 +267,7 @@ async fn unlock_wallet<S: EthereumSigner + Clone>(
             .send()
             .await
             .map_err(|e| GenericError::new(format!("Failed to send change_pubkey request: '{}'. HINT: Did you run `yagna payment fund` and follow the instructions?", e)))?;
-        log::debug!("Unlock tx: {:?}", unlock);
+        // DO WE NEED DEBUG? log::debug!("Unlock tx: {:?}", unlock);
```

Comment:
I'd suggest `log::trace()` for this.

Reply:
made it a cleaner message, so kept it as debug.. please check!

Reply:
seems trash. Pls resolve it

Reply:
as i said i made it a better message, seems the update got missing... checking

Reply:
seems all info is printed as needed, so removed the ugly line on `payments/beta-1` branch